app/models/application_state.rb

```diff
@@ -4,6 +4,9 @@ class ApplicationState < ApplicationRecord
   belongs_to :application
   belongs_to :lead_provider, optional: true
 
+  # TODO: remove "allow_nil" and add default value "gen_random_uuid()" and constraints into the DB after separation
```

Comment:
this isn't surface in the API, so can allow it to remain `nil` after separation and delete that field as well